### ISSUE TITLE
Fonts with comma in filename break on CLI

### ISSUE THREAD

**Issue by SvanteRichter** [maplibre/font-maker#38]
Hello!

I was using the CLI to convert google fonts and it seems like it cannot handle files with commas like for example the upstream google font "NotoSans[wdth,wght].ttf". It seems like the web version works fine.

If I just rename the font to not include a comma it works.

Thanks!

## How to reproduce
* Build the CLI
* Download a font with a comma in it (example: https://github.com/google/fonts/blob/main/ofl/notosans/NotoSans%5Bwdth%2Cwght%5D.ttf)
* Run font-maker `./font-maker --name "NotoSans" "./tmp" "./NotoSans[wdth,wght].ttf"`
* Get the error:
```txt
Adding ./NotoSans[wdth
terminate called after throwing an instance of 'std::runtime_error'
  what():  Could not open font face
Aborted (core dumped)
```

## Attachments
- [x] Include the font file (.ttf or .otf) you are trying to convert. 
[NotoSans\[wdth,wght\].zip](https://github.com/user-attachments/files/18148075/NotoSans.wdth.wght.zip)

**Comment by bdon** [comment]
Looks like a limitation of cxxopts: https://github.com/jarro2783/cxxopts/issues/228

You can add `#define CXXOPTS_VECTOR_DELIMITER '\0'` to the top of main.cpp as a workaround

Or maybe we should just require the input doesn't have commas?

**Comment by bdon** [comment]
Should be fixed in #39 , thanks.
